libs/types/src/tokens.rs

```diff
@@ -59,6 +60,39 @@ impl From<u32> for CurrencyId {
 	}
 }
 
+impl TryInto<u128> for CurrencyId {
+	type Error = DispatchError;
+
+	// TODO: Fix dummy values
+	fn try_into(self) -> Result<u128, Self::Error> {
+		match self {
+			// [0, ..., 0, 0]
+			CurrencyId::Native => Ok(u128::from_be_bytes([0u8; 16])),
+			CurrencyId::Tranche(_, _) => Err(DispatchError::Token(TokenError::Unsupported)),
+			// [0, ..., 0, 1]
+			CurrencyId::KSM => {
+				let mut r = [0u8; 16];
+				r[15] = 1;
+				Ok(u128::from_be_bytes(r))
+			}
+			// [0, ..., 0, 2]
+			CurrencyId::AUSD => {
+				let mut r = [0u8; 16];
+				r[15] = 2;
+				Ok(u128::from_be_bytes(r))
+			}
```

Comment:
`KSM` and `AUSD` will be dropped so in this conversion we can already ignore those two. And I still think that having a storage for this would grant us more granular control over what tokens we allow to be Connector-transferrable and to make (reverse-)maping more obvious than an hardcoded runtime feature IMHO (I can be wrong in this last one and happy to change my mind)

Reply:
If we do keep this logic, we need testing covering this 👀